hyperkube/Makefile

```diff
@@ -70,6 +69,7 @@ $(BINARY): $(BINARY).tmp master-multi.json.build master.json.build safe_format_a
 
 $(BINARY).tmp: $(KUBEROOT)/_output/dockerized/bin/linux/amd64/$(BINARY)
 	cp $(KUBEROOT)/_output/dockerized/bin/linux/amd64/$(BINARY) $(BINARY).tmp
+	cp $(KUBEROOT)/_output/dockerized/bin/linux/amd64/kubectl kubectl.tmp
```

Comment:
all: $(BINARY) kubectl


so also above: KUBECTL=kubectl
then use $(KUBECTL) instead of kubectl everywhere